entrypoint.sh

```diff
@@ -118,38 +119,27 @@ cleanup() {
 
 trap 'cleanup $?' EXIT
 
-if ! is_installed lhci; then
```

Comment:
I can't imagine any case when `lhci` and `shopify` are **not** already installed, as this `entrypoint.sh` script is based on a `Dockerfile`, in which both tools are installed.

Moreover, if you take a look at the `Dockerfile`, `@lhci/cli@0.8.x` is installed, so the command here is even outdated.
As I was initially confused and changing the command in _this_ `entrypoint.sh` file, I decided to remove these lines.